src/sleepwellbaby/explainability.py

```diff
@@ -0,0 +1,46 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import shap
+import sklearn
+
+
+def get_shap_values(model, X):
+    if isinstance(model, sklearn.calibration.CalibratedClassifierCV):
```

Comment:
It's cleaner to check if it is not an instance of calibratedClassifierCV and if so raise an error, that way the rest of the function doesn't need to be indented.